templates/time_tracking_entry.tpl.html

```diff
@@ -79,58 +88,70 @@
     }
   });
 
+{if $details|default:'' == ''}
   Eventum.updateTimeFields($form, 'date[Year]', 'date[Month]', 'date[Day]', 'date[Hour]', 'date[Minute]');
   Eventum.updateTimeFields($form, 'date2[Year]', 'date2[Month]', 'date2[Day]', 'date2[Hour]', 'date2[Minute]');
+{/if}
   $form.submit(validateTimeForm);
 });
 //-->
 </script>
 
     <form id="add_time_form" method="post">
+        {if $details|default:'' != ''}
+        <input type="hidden" name="cat" value="update_time">
+        <input type="hidden" name="ttr_id" value="{$details.ttr_id}">
+        {else}
         <input type="hidden" name="cat" value="add_time">
         <input type="hidden" name="issue_id" value="{$issue_id}">
+        {/if}
         <table class="bordered">
             {if $core.current_role > $core.roles.viewer}
             <tr class="title">
-                <th colspan="2">{t}Record Time Worked{/t}</th>
+                <th colspan="2">{t 1=$action}%1 Time Worked{/t}</th>
```

Comment:
this is not translatable, make two different `{t}` placeholders. assuming that word order in sentence is same in all languages is naïve